web/src/pages/WelcomePage.css

```diff
@@ -0,0 +1,21 @@
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    flex-wrap: wrap;
+    margin-bottom: 20px;
+}
+.box { 
+        width: 300px;
+        height: 200px;
+        background-color: lightblue;
+        opacity: 90%;
+        border: 1px solid gray;
+        margin: 0,10px;
```

Comment:
this looks like a syntax error. Maybe you meant `margin: 0 10px;`?

Reply:
removed that line, because it doesn't seem to make any difference.